apps/sv/frontend/src/__tests__/governance/proposal-details-content.test.tsx

```diff
@@ -96,19 +110,71 @@ const voteResult = {
   ],
 } as ProposalDetailsContentProps;
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchInterval: 500,
+      structuralSharing: replaceEqualDeep,
+    },
+  },
+});
+
 const Wrapper: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+  return (
+    <MemoryRouter>
+      <SvConfigProvider>
+        <WrapperProviders children={children} />
+      </SvConfigProvider>
+    </MemoryRouter>
+  );
+};
+
+const WrapperProviders: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+  const config = useSvConfig();
+  const navigate = useNavigate();
+
   return (
     <ThemeProvider theme={theme}>
-      <MemoryRouter>{children}</MemoryRouter>
+      <AuthProvider authConf={config.auth} redirect={(path: string) => navigate(path)}>
+        <QueryClientProvider client={queryClient}>
+          <UserProvider authConf={config.auth} testAuthConf={config.testAuth}>
+            <SvClientProvider url={config.services.sv.url}>
+              <SvAdminClientProvider url={config.services.sv.url}>{children}</SvAdminClientProvider>
+            </SvClientProvider>
+          </UserProvider>
+        </QueryClientProvider>
+      </AuthProvider>
     </ThemeProvider>
   );
 };
 
+describe('SV user can', () => {
```

Comment:
I actually don't care about this test here but need it to setup my auth access token which i need when voting.